### ISSUE TITLE
Random runtime errors 

### ISSUE THREAD

**Issue by exaroth** [erroneousboat/slack-term#62]
Hi, great app, however I noticed that randomly (and quite often) slack-term crashes with following runtime err:
```
goroutine 8 [running]: 
github.com/erroneousboat/slack-term/vendor/github.com/nsf/termbox-go.PollEvent(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
/Users/konrad/.go/src/github.com/erroneousboat/slack-term/vendor/github.com/nsf/termbox-go/api.go:330 +0x68a
github.com/erroneousboat/slack-term/vendor/github.com/gizak/termui.hookTermboxEvt()                                                                                                                        
/Users/konrad/.go/src/github.com/erroneousboat/slackterm/vendor/github.com/gizak/termui/events.go:126 +0xa3                                                                                       │
created by github.com/erroneousboat/slack-term/vendor/github.com/gizak/termui.Init                                                                                                                         /Users/konrad/.go/src/github.com/erroneousboat/slack-term/vendor/github.com/gizak/termui/render.go:38 +0xa4
```

I'm using version 0.2.2 on zsh running tmux (or rather tmux running zsh).

**Comment by sholmesy** [comment]
Had a similar issue:

```
github.com/erroneousboat/slack-term/vendor/github.com/nsf/termbox-go.PollEvent(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)        │
term/vendor/github.com/nsf/termbox-go/api.go:330 +0x44f────────┘
github.com/erroneousboat/slack-term/handlers.eventHandler.func1(0xc420316c40)
	/Users/xxx/go/src/github.com/erroneousboat/slack-term/handlers/event.go:50 +0x30
created by github.com/erroneousboat/slack-term/handlers.eventHandler
	/Users/xxx/go/src/github.com/erroneousboat/slack-term/handlers/event.go:52 +0x3f```

**Comment by erroneousboat** [comment]
Duplicate of: #47 and #58. Closing this one, changing title of #58.

Hopefully with the new v0.2.3 release this has been solved, I'll keep #58 open for further discussion/feedback.
